app/connectors/connector-verifier/src/main/java/io/syndesis/verifier/api/SimpleComponentVerifier.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.verifier.api;
+
+import java.util.Optional;
+
+import org.apache.camel.component.extension.ComponentVerifierExtension;
+
+public class SimpleComponentVerifier extends BaseVerifier {
```

Comment:
Not sure whether we need this in addition to the BaseVerifier (if we put the constructor and a default implementation of getConnectorAction() into the BaseVerifier, too)

Reply:
I think this class is a consequence to have the BaseVerifier abstract, I can get rid of it once BaseVerifier is not more abstract